scripts/ec2/release_address.coffee

```diff
@@ -0,0 +1,22 @@
+# Description:
+#   Release elastic IP's
+#
+# Commands:
+#   hubot ec2 release address {eipalloc-id}
+#   hubot ec2 release address {publicIp}
+
+aws = require('../../aws').aws()
+ec2 = new aws.EC2()
+
+module.exports = (robot) ->
+  robot.respond /ec2 release address (.*)$/i, (msg) ->
+    params = {}
+    if msg.match[1].split('.').length == 4
+      params.PublicIP = msg.match[1]
+    else if msg.match[1].match(/^eipalloc-[a-zA-Z0-9].*/i)
+      params.AllocationId = msg.match[1]
+    else
+      return msg.send 'I need the Public IP or allocation ID to release an elastic IP.'
+
+    ec2.releaseAddress params, (response) ->
+      msg.send response
```

Comment:
It works well, but the return message(null) might cause to be uneasy for users.

```
hubot> hubot ec2 release address eipalloc-860848e3
hubot> null
```

How about this diff for example ? (see https://github.com/yoheimuta/hubot-aws/blob/f872d48648dfd4bec94c74eeb0759e7a198548c3/scripts/ec2/delete_tags.coffee#L64)

```diff
+ msg.send "Success to release an address"
msg.send response
```